tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -151,7 +151,7 @@ class WskBasicUsageTests
         val tmpwskprops = File.createTempFile("wskprops", ".tmp")
         try {
             val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
-            val apihost = s"http://${WhiskProperties.getBaseControllerAddress()}"
+            val apihost = s"${WhiskProperties.getBaseControllerAddress()}"
```

Comment:
@lzbj, also looks like the test failed because the server is expecting an http request instead of an https request.